provision/acc_provision/acc_provision.py

```diff
@@ -1345,14 +1361,17 @@ def generate_calico_deployment_files(args, config, network_operator_output):
         custom_resources_calicoctl_yaml = calico_bgp_config_output + bgp_peer + bgp_node
 
         if args.cko:
+            netopConfig = dict(config)
+            if "connectivity_checker" in netopConfig.keys():
```

Comment:
If we remove this check here, then all cases (with cko arg and without connectivity_check) it will try to generate nettools deployment files which will fail because of nettools config does not exist. So we need this check here.